libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -19,12 +19,11 @@
 
 #include <AP_HAL/AP_HAL.h>
 #include <AP_RPM/AP_RPM.h>
-#include <AP_AHRS/AP_AHRS.h>
```

Comment:
There's no direct include of this anymore, which means we're relying on transitive includes to get `AP::ahrs()` in the `.cpp` now.

Probably a good idea to move this `#include` into the `.cpp` rather than removing it entirely.